Stratis.Bitcoin/Configuration/NodeSettings.cs

```diff
@@ -109,6 +111,7 @@ public NodeSettings()
 
 			nodeSettings.RequireStandard = config.GetOrDefault("acceptnonstdtxn", !(nodeSettings.RegTest || nodeSettings.Testnet));
 			nodeSettings.MaxTipAge = config.GetOrDefault("maxtipage", DEFAULT_MAX_TIP_AGE);
+			nodeSettings.ApiUri = config.GetOrDefault("apiuri", new Uri("http://localhost:5000"));
```

Comment:
Don't we prefer to use the port only? is there a case where not localhost will be used?